tests/test_svs_disp.py

```diff
@@ -0,0 +1,28 @@
+import subprocess
+import unittest
+
+from base import SD_VM_Local_Test
+
+
+class SD_SVS_Disp_Tests(SD_VM_Local_Test):
+    def setUp(self):
+        self.vm_name = "sd-svs-disp"
+        super(SD_SVS_Disp_Tests, self).setUp()
+
+    def test_sd_client_package_installed(self):
+        pkg = "securedrop-workstation-svs-disp"
+        self.assertTrue(self._package_is_installed(pkg))
```

Comment:
Great catch on re-enabling these tests! Some of the test names still refer to `sd-svs` (specifically `test_sd_client_package_installed`), will clean up for clarity.